components/articles/cards/ArticleCard.js

```diff
@@ -68,6 +68,10 @@ const getCoverLink = (images, cardSize) => {
 
 const COLLECTION_LOGO_WIDTH = 70;
 
+const COLLECTIONS_HIDE_COVER = {
+  starbellit: true,
+};
```

Comment:
is there any cleaner solution?

Reply:
I guess there is a way to use css only and somehow resolve long titles there, but I don't really want to do that.